languages/java/exercises/concept/strings/.docs/hints.md

```diff
@@ -0,0 +1,23 @@
+## General
+
+- The `string` class has many useful [built-in methods][string-class].
+
+## 1. Get message from a log line
+
+- Different options to search for text in a string are explored in [this tutorial][tutorial-search-text-in-string].
+- Removing white space is [built-in][tutorial-trim-white-space].
+
+## 2. Get log level from a log line
+
+- Changing a `string`'s casing is explored [here][tutorial-changing-case-upper] and [here][tutorial-changing-case-lower].
+
+## 3. Reformat a log line
+
+- There are several ways to [concatenate strings][tutorial-concatenate-strings]
```

Comment:
```suggestion
- There are several ways to [concatenate Strings][tutorial-concatenate-strings]
```

Reply:
I am not sure about this suggestion. 
Here strings is about the word plural string (intended as natural language).
If I start it with the capital letter it seems exist in Java a class o something relative to code named Strings and this is not the truth.